src/components/PopoverMenu/index.native.js

```diff
@@ -28,6 +28,7 @@ class PopoverMenu extends Component {
             <BasePopoverMenu
                 // eslint-disable-next-line react/jsx-props-no-spreading
                 {...this.props}
+                allowKeyboardNavigation={false}
```

Comment:
This makes the code harder to read. This is a very bad example of using props. 
You are setting the same prop to true on native and false on the web. Although this should never work on natives.
```suggestion
```

Reply:
I didn't understand. 

> You are setting the same prop to true on native and false on the web

I am setting it false on `index.native.js` and user-driven for `index.js`. This is to ensure that if the developer sets the prop by mistake it overrides it to `false`

Reply:
That is misleading. we should not do that. This prop should not have an effect on natives at all even if the developer sets it true on natives.